### DISCUSSION TITLE
problema para encontrar "server"

### DISCUSSION THREAD

**Discussion by polyanw** [balta-io/7188#28] [Perguntas e Respostas] [answered]
na nova versão do mockoon não achei import all environments from file, e não sei onde estaria o destino 7188 > api > servers (no caso da aula está na pasta cursos, mas não me recordo de termos criado uma pasta para este fim específico...)

agradeço desde já!
![image](https://user-images.githubusercontent.com/103971193/166345745-145f458d-c61d-40e7-9bfd-78233990c358.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @polyanw, basta seguir este processo:
https://github.com/balta-io/7187/discussions/60#discussioncomment-2372392
